src/main/java/pl/javastart/task/Main.java

```diff
@@ -3,6 +3,18 @@
 public class Main {
 
     public static void main(String[] args) {
+        String product1Name, product2Name, product1Company, product2Company;
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu. W instrukcji jest też informacja jak ustawić dla wszystkich nowych projektów

Reply:
nie definiuj kilku pól w jednej linii